docs/tutorials/getting_started.md

```diff
@@ -1,12 +1,15 @@
 # Getting started
 
-In this tutorial, we will deploy and run the SD-Core 5G core network using Juju. We will also
-deploy a radio and cellphone simulator to simulate usage of this network.
+In this tutorial, we will deploy and run the SD-Core 5G core network using Juju and Terraform.
+As part of this tutorial, we will also deploy additional components (gNB Simulator - a 5G radio 
+and a cellphone simulator, SD-Core Router - a software router facilitating communication between
+the core and the Radio Access Network (RAN)) to simulate usage of this network. Both gNB Simulator 
+and SD-Core Router serve only demonstration purposes and shouldn't be part of production 
+deployments.
 
-To complete this tutorial, you will need a machine with the following
-requirements:
+To complete this tutorial, you will need a machine which meets the following requirements:
 
-- A recent `x86_64` CPU (Intel 4ᵗʰ generation or newer, or AMD Ryzen or newer)
+- A recent `x86_64` CPU (Intel 4ᵗʰ generation or newer or AMD Ryzen or newer)
```

Comment:
I would suggest the comma stays, as it is separating the grouping of "Intel or newer" from "AMD or newer"

Also, I just realized we did not specify the number of CPUs here, but that could be a different PR

Reply:
Done.
Also added info about required number of cores.